context/app/static/js/components/entity-search/Search/Search.jsx

```diff
@@ -55,19 +57,20 @@ function Search({ numericFacetsProps }) {
       facets: Object.values(facets).map((facet) =>
         createSearchkitFacet({ ...facet, ...getRangeProps(facet.field, numericFacetsProps) }),
       ),
-      filters: filters.map((filter) => filter.definition),
+      filters: defaultFilterValues.map((filter) => filter.definition),
     }),
-    [authHeader, elasticsearchEndpoint, facets, fields, filters, numericFacetsProps],
+    [authHeader, defaultFilterValues, elasticsearchEndpoint, facets, fields, numericFacetsProps],
   );
 
   const transporter = new RequestTransporter(config);
 
   const variables = useSearchkitVariables();
-  const { results } = useSearchkitSDK(config, variables, transporter, filters);
+  const { results } = useSearchkitSDK(config, variables, transporter, defaultFilterValues);
```

Comment:
I'm a little confused... The user-specified filters wouldn't go in here? Where are they?

Reply:
All facet filters are managed internally by searchkit.